### PR TITLE
enhancement for cve security notification 

### PR DESCRIPTION
**Pull request by nurali-techie**
Minor leftover from PR https://github.com/fabric8-services/fabric8-notification/pull/75
Now consider codebase.cve-scan attribute.

Fix - https://openshift.io/openshiftio/openshiftio/plan/detail/2647

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @nurali-techie for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

File: collector/test-files/cve/codebase.search.testproject1.json [hunk 1]
```diff
@@ -1,275 +1,309 @@
...
+            "self": "http://127.0.0.1:8080/api/codebases/9fb1065d-8d25-42b3-bd62-01dd23c225c9/workspaces"
+          }
+        }
+      },
+      "type": "codebases"
+    },
+    {
+      "attributes": {
+        "createdAt": "2018-07-03T13:01:03.521885Z",
+        "cve-scan": false,
```
Comment (nurali-techie) [inline collector/test-files/cve/codebase.search.testproject1.json]: one codebase test-data added with cve-scan=false

File: glide.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ package: github.com/fabric8-services/fabric8-notification
 license: Apache-2.0
 import:
 - package: github.com/fabric8-services/fabric8-wit
-  version: 8e74c765391d51a548ffb351038c32fe932398d9
+  version: 88b4ed94f58e2455953ee4c452e07e12b4a087c2
```
Comment (nurali-techie) [inline glide.yaml]: point to latest wit/master version.

File: template/analytics.notify.cve/body.html [hunk 3]
```diff
@@ -106,7 +106,16 @@
...
+          <td>
+            {{ $cvesArr := $dependency.cves }}
+            {{- range $ind, $cveMap := $cvesArr -}}
+              {{ if (eq $ind 0) }}
+                {{ $cveMap.CVE }} <b>[{{ $cveMap.CVSS }}/10]</b>
+              {{- else -}}
+                ,&nbsp;{{ $cveMap.CVE }} <b>[{{ $cveMap.CVSS }}/10]</b>
+              {{- end -}}
+            {{ end }}
+          </td>
```
Comment (nurali-techie) [inline template/analytics.notify.cve/body.html]: rectify issue from preview PR - https://github.com/fabric8-services/fabric8-notification/pull/75/files#r200899846

File: collector/cve.go [hunk 4]
```diff
@@ -33,9 +33,11 @@ func NewCVEResolver(authClient *authapi.Client, witClient *api.Client) ReceiverR
 func collectCodebasesSpaces(codebases *api.CodebaseList) []uuid.UUID {
 	var spaces []uuid.UUID
 	for _, cb := range codebases.Data {
-		space, err := uuid.FromString(*cb.Relationships.Space.Data.ID)
-		if err == nil {
-			spaces = append(spaces, space)
+		if *cb.Attributes.CveScan {
```
Comment (aslakknutsen) [inline collector/cve.go]: guard against nil, nil is false in our case

**Pull request merged**